gen-app-builder/data-store-status-checker/examples/data_store_checker.ipynb

```diff
@@ -0,0 +1,433 @@
+        "docs = list_documents(project_id, location, datastore_id)\n",
+        "\n",
+        "if len(docs) == 0:\n",
+        "    print(PENDING_MESSAGE)\n",
+        "else:\n",
+        "    SUCCESS_MESSAGE = f\"\"\"\n",
+        "  Success! 🎉\\n\n",
+        "  Your indexing is complete.\\n\n",
+        "  Your index contains {len(docs)} documents.\n",
+        "  \"\"\"\n",
+        "    print(SUCCESS_MESSAGE)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "f957771b340a"
+      },
+      "source": [
+        "# List Documents\n",
+        "List all the documents for a given Data Store ID"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "1fa7c648a7ae"
+      },
+      "outputs": [],
+      "source": [
+        "docs = list_documents(project_id, location, datastore_id)\n",
+        "docs[0]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "efed4f2ce2e1"
+      },
+      "source": [
+        "# Search Data Store by Doc ID\n",
+        "Search through all Docs in a given Data Store and find a specific Doc ID."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "f86d23b8f750"
+      },
+      "outputs": [],
+      "source": [
+        "document_id = \"a71d802406f2f0e546b621245e1cbc6a\"\n",
+        "\n",
+        "search_doc_id(document_id, docs)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "4b72dae5ea9f"
+      },
+      "source": [
+        "# List Indexed URLs"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "f04f01d4a9f2"
+      },
+      "outputs": [],
+      "source": [
+        "urls = list_indexed_urls(project_id, location, datastore_id, docs)\n",
+        "urls[0]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "6323e4d6e124"
+      },
+      "source": [
+        "# Search Indexed URLs"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "id": "1fb7f53965ab"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "https://www.yeti.com/bags/luggage/26010000103.html?bvstate=pg:2/ct:r\n",
+            "https://www.yeti.com/bags/luggage/crossroads-luggage-29in.html\n",
+            "https://www.yeti.com/bags/luggage/18060131085.html?bvstate=pg:2/ct:r\n",
+            "https://www.yeti.com/bags/luggage/crossroads-luggage-29in.html?bvstate=pg:2/ct:r\n",
+            "https://www.yeti.com/bags/luggage/crossroads-luggage-22in.html\n",
+            "https://www.yeti.com/bags/luggage\n"
+          ]
+        }
+      ],
+      "source": [
+        "search_url(urls, \"https://www.yeti.com/bags/luggage\")"
```

Comment:
Just for this example notebook input/output, would it make sense to generalize these URLs (e.g., `example.com` or point to a Google-owned domain? Not sure how permissions/licensing works when referring to third-party companies like this in our examples. 🤷

Reply:
Ack and good point.  
I've updated all of the content to reflect a `cloud.google.com` example instead.